docs/user-guide/configuration.md

```diff
@@ -99,6 +99,7 @@ settings:
   default: true
   description: false
   escape: true
+  hideempty: false
```

Comment:
This should be `hide-empty`

Reply:
Oops apologies! You were clear in your previous comment https://github.com/terraform-docs/terraform-docs/pull/551#issuecomment-908654837. But for some reason  I saw `lockfile` down below and I figured that i had to remove the dash!

Just addressed in ffe84881adb239267543c852fa5e17ec14a3e279.

Reply:
It's alright, no worries 🙂